frontend/cypress/e2e/0-osmose-tests/login.cy.js

```diff
@@ -0,0 +1,14 @@
+describe('login page', () => {
+  beforeEach(() => {
+    cy.visit('http://localhost:3000/')
+  })
+
+  it('logs in with right credentials', () => {
+    cy.get("#loginInput").click();
+    cy.get("#loginInput").type("admin");
+    cy.get("#passwordInput").click();
+    cy.get("#passwordInput").type("osmose29");
+    cy.get("form > input").click();
+    cy.get("h1").should('have.text', 'APLOSEDatasets')
```

Comment:
I'm pretty sure there is an error in the text

Reply:
On that page we have two H1 titles, which is not a great idea and we should fix. There is a main H1 APLOSE title and then a second page-specific H1 Datasets title. When using `get=>have.text` you get the text of all the hits, so here `APLOSEDatasets`. We could try to get the second h1 only but a better long term change is removing the H1 title or moving page-titles to H2.